roles/custom/matrix-synapse-reverse-proxy-companion/templates/labels.j2

```diff
@@ -119,6 +119,44 @@ traefik.http.routers.matrix-synapse-reverse-proxy-companion-public-client-synaps
 ############################################################
 {% endif %}
 
+{% if matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_enabled %}
+############################################################
+#                                                          #
+# Internal Synapse Admin API (/_synapse/client)            #
+#                                                          #
+############################################################
+
+traefik.http.routers.matrix-synapse-reverse-proxy-companion-internal-client-synapse-client-api.rule=PathPrefix(`/_synapse/client`)
```

Comment:
It'd be better if the rule is coming from a variable (`matrix_synapse_reverse_proxy_companion_container_labels_internal_client_synapse_admin_api_traefik_rule`), defined in `default/main.yml`  in the `matrix-synapse-reverse-proxy-companion` role.

Here's an example: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/e000cbf0f42874bec05611d57a2922259da292bc/roles/custom/matrix-synapse-reverse-proxy-companion/defaults/main.yml#L78

You may even consider adding a `_priority` variable ([like this](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/e000cbf0f42874bec05611d57a2922259da292bc/roles/custom/matrix-synapse-reverse-proxy-companion/defaults/main.yml#L79)), getting its value from the `matrix-synapse` role. In the `matrix-synapse` role, a priority of `0` is a good default, I suppose.